cypress/e2e/prompt_playground/provider.py

```diff
@@ -71,5 +73,24 @@ async def create_event_stream():
     is_chat=True,
 )
 
+LangchainTestLLM = LangchainGenericProvider(
+    id="test-langchain",
+    name="TestLangchain",
+    llm=None,
+    env_vars={"api_key": "TEST_LLM_API_KEY"},
+    inputs=[
+        Select(
+            id="model",
+            label="Model",
+            values=["test-model-langchain-1", "test-model-langchain-2"],
+            initial_value="test-model-langchain-2",
+        ),
+    ],
+    is_chat=False,
+    is_automated=False,
+)
+LangchainTestLLM.create_completion = ChatTestLLM.create_completion
```

Comment:
Why are we not passing the llm class directly?

Reply:
right the langchain `FakeListLLM` seems like a good choice there 👌